.changeset/swift-cows-thank.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog': patch
+---
+
+Removed fullScreen property from UnregisterEntity Dialog modal.
```

Comment:
This is totally fine, but just wanted to throw this out there:
https://chris.beams.io/posts/git-commit/#seven-rules

This patch says _how_ you did it, but not really the what or intent around it?  No need to change it now it's good, but going forward something descriptive like "Fixed Unregister Entity popup from going full screen at smaller screen sizes" I think might be preferred, and I can just click the PR or commit to see clearly that how you fixed it was by removing the fullScreen property.  Just a casual note!

Reply:
Oh! Thanks for letting me know. Much appreciated!